source/module_base/clebsch_gordan_coeff.cpp

```diff
@@ -30,6 +31,7 @@ void Clebsch_Gordan::clebsch_gordan(const int& lli,
     ModuleBase::matrix ylm(llx, llx);
     ModuleBase::matrix mly(llx, llx);
 
+    ModuleBase::Memory::record("Clebsch_Gordan::matrix",sizeof(double)*llx*llx*2+sizeof(ModuleBase::Vector3<double>)*llx);
```

Comment:
what size will `llx` be in your test?
Only significant should be recorded.

Reply:
Yes, there are some very small parts, I will change to only record the significant parts.